package.json

```diff
@@ -126,7 +128,9 @@
   "dependencies": {
     "@kadira/storybook": "^1.36.0",
     "classnames": "^2.2.5",
+    "commander": "^2.11.0",
```

Comment:
We'll want these deps in dev dependencies.

Reply:
I went back and forth on this one. ok